bsonnumpy/bson/bson-iter.c

```diff
@@ -737,8 +831,6 @@ _bson_iter_next_internal (bson_iter_t *iter,   /* INOUT */
    return false;
 }
 
-
-/* added to BSON-NumPy's vendored libbson */
```

Comment:
We should keep this comment so that future readers know this method was added. Also, perhaps we can refactor this method to use libbson's `_bson_iter_next_internal` method now that it accepts a `next_keylen` argument. Can you open a new issue to investigate this?